modules/core/src/lib/deck.js

```diff
@@ -499,7 +499,9 @@ export default class Deck {
     this.effectManager = new EffectManager({gl, layerManager: this.layerManager});
 
     for (const effect of this.props.effects) {
-      this.effectManager.addEffect(effect);
+      if (effect instanceof Effect) {
```

Comment:
- This for loop should be inside of the `EffectManager`.
- It should be possible to update `effects` via `setProps`.

Reply:
yeah, will refactor this part after Effect RFC.